include/aos/common/crypto.hpp

```diff
@@ -358,7 +358,7 @@ struct Certificate {
      */
     StaticArray<uint8_t, cCertIssuerSize> mIssuer;
     /**
-     * DER encoded certificate serial number.
+     * Certificate serial number.
```

Comment:
Actually you should put message body as commit header. It is self-explanatory,

Reply:
Added additional comment concerning Certificate::mSerial type.